### PR TITLE
Feature/grades

### PR DESCRIPTION
**Pull request by jessica**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "react-relay": "^8.0.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.3.1",
+    "react-window": "^1.8.6",
```
Comment (HeIp) [inline frontend/package.json:16]: this package was added but never used ?

File: frontend/yarn.lock [hunk 2]
```diff
@@ -7017,6 +7017,11 @@ mem@^4.0.0:
     mimic-fn "^2.0.0"
     p-is-promise "^2.0.0"
 
+"memoize-one@>=3.1.1 <6":
+  version "5.1.1"
+  resolved "https://registry.yarnpkg.com/memoize-one/-/memoize-one-5.1.1.tgz#047b6e3199b508eaec03504de71229b8eb1d75c0"
+  integrity sha512-HKeeBpWvqiVJD57ZUAsJNm71eHTykffzcLZVYWiVfQeI1rJtuEaS7hQiEpWfVVk18donPwJEcFKIkCmPJNOhHA==
```
Comment (HeIp) [inline frontend/yarn.lock:7023]: this package was added but never used as well?

File: frontend/yarn.lock [hunk 3]
```diff
@@ -9555,6 +9560,14 @@ react-scripts@3.3.1:
...
     fsevents "2.1.2"
 
+react-window@^1.8.6:
+  version "1.8.6"
+  resolved "https://registry.yarnpkg.com/react-window/-/react-window-1.8.6.tgz#d011950ac643a994118632665aad0c6382e2a112"
+  integrity sha512-8VwEEYyjz6DCnGBsd+MgkD0KJ2/OXFULyDtorIiTz+QzwoP94tBoA7CnbtyXMm+cCeAUER5KJcPtWl9cpKbOBg==
+  dependencies:
+    "@babel/runtime" "^7.0.0"
+    memoize-one ">=3.1.1 <6"
+
```
Comment (HeIp) [inline frontend/yarn.lock:9570]: this package was added but never used as well?

**Review by HeIp — CHANGES_REQUESTED** [review]
Needs to look over some things

File: frontend/package.json (hunk 1 above)
Comment (jessica) [reply frontend/package.json:16]: I added it when I was trying to do the instructor grade book table. I forgot to remove it afterwards

Comment (HeIp) [reply frontend/package.json:16]: Should update the branch to remove these package changes

**Review by HeIp — CHANGES_REQUESTED** [review]
Just want to confirm you didn't manually edit these and removed them with terminal/cli

**Comment by jessica** [comment]
> Just want to confirm you didn't manually edit these and removed them with terminal/cli

I removed it using yarn remove react-window

**Review by cbarb — APPROVED** [review]

**Review by HeIp — APPROVED** [review]
Ready to merge

**Pull request merged**